packages/runtime/src/runtime/hooks/useEnsembleAction.tsx

```diff
@@ -141,6 +144,7 @@ export const useExecuteCode: EnsembleActionHook<
             {
               ...customWidgets,
               ensemble: {
+                ...themescope,
```

Comment:
Just need the setTheme method here?

Reply:
you are right, but in case if we need `ensemble.themeName` or `ensemble.theme` in any case, then this will be useful